actors/runtime/indices/indices.go

```diff
@@ -365,6 +365,12 @@ func StorageMining_SpcLookbackSeal() abi.ChainEpoch {
 	return StorageMining_Finality() // should be approximately the same as finality
 }
 
+func StorageMining_SpcLookbackSealLimit() abi.ChainEpoch {
+	PARAM_FINISH()
+	return 0
+}
+
+
```

Comment:
I don't think this is necessary. The point is that number is derived from the maxSealTime below. Happy to chat through it @zixuanzh